src/models/auto_clm.py

```diff
@@ -52,6 +54,9 @@ def get_auto_clm_tokenizer(
     overwatch.info(f"Fetching Hugging Face [Fast] AutoTokenizer for Model: `{REGISTRY[model_id]}`...")
     if use_pretrained_tokenizer:
         tokenizer = AutoTokenizer.from_pretrained(REGISTRY[model_id], config=config, cache_dir=paths["tokenizer"])
+    elif use_passthrough_tokenizer:
```

Comment:
could we add an assert that at most one of use_pretrained_tokenizer and use_passthrough_tokenizer are set?

Reply:
Good idea. I added this.